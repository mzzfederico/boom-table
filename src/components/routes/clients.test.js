/* eslint-disable no-undef */
import React from "react";
import Clients from "./index";
import renderer from "react-test-renderer";

it("renders without crashing", () => {
    const tree = renderer.create(
        <Clients />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});