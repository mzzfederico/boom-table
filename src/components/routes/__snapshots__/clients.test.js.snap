// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
Array [
  <nav>
    <ul>
      <li>
        <a
          aria-current="page"
          className=""
          href="/"
          onClick={[Function]}
          style={Object {}}
        >
          Summary
        </a>
      </li>
      <li>
        <a
          aria-current={null}
          href="/clients"
          onClick={[Function]}
        >
          Clients
        </a>
      </li>
      <li>
        <a
          aria-current={null}
          href="/photoshoots"
          onClick={[Function]}
        >
          Photoshoots
        </a>
      </li>
      <li>
        <a
          aria-current={null}
          href="/typologies"
          onClick={[Function]}
        >
          Typologies
        </a>
      </li>
    </ul>
    <style
      jsx={true}
    >
      
            nav {
                margin-bottom: 1rem;
            }
            ul {
                list-style: none;
                margin: 0;
                padding: 0;
            }
            li {
                display: inline;
            }
            li:nth-child(1) {
                margin-right: 1rem;
            }
            li a {
                padding: 0.25rem;
                color: black;
                text-decoration: none;
            }
            li a.active {
                border-bottom: 2px solid black;
            }
        
    </style>
  </nav>,
  <div
    className="WeekTable"
  >
    <div
      className="column column-clients"
    >
      clients
      <style
        jsx={true}
      >
        
                .column {
                    text-transform: lowercase;
                    font-variant: small-caps;
                    font-weight: 500;
                    font-size: 0.85rem;
                    border-bottom: 1px solid #f7f7f7;
                    margin-bottom: 1rem;
                }

                .column-clients {
                    grid-column: 1;
                }
            
      </style>
    </div>
    <div
      className="column column-monday"
    >
      monday
      <style
        jsx={true}
      >
        
                .column {
                    text-transform: lowercase;
                    font-variant: small-caps;
                    font-weight: 500;
                    font-size: 0.85rem;
                    border-bottom: 1px solid #f7f7f7;
                    margin-bottom: 1rem;
                }

                .column-monday {
                    grid-column: 2;
                }
            
      </style>
    </div>
    <div
      className="client client-monday total-bottom"
    >
      <div
        className="number_of_photos"
      >
        -
      </div>
      <style
        jsx={true}
      >
        
            .client-monday {
                grid-column: 2
            }

            .client p {
                font-size: 0.8rem;
                margin: 0;
            }

            .client p span {
                font-size: 1rem;
                font-weight: 700;
            }

            .number_of_photos span:nth-child(2) {
                font-size: 0.7rem;
            }
        
      </style>
    </div>
    <div
      className="column column-tuesday"
    >
      tuesday
      <style
        jsx={true}
      >
        
                .column {
                    text-transform: lowercase;
                    font-variant: small-caps;
                    font-weight: 500;
                    font-size: 0.85rem;
                    border-bottom: 1px solid #f7f7f7;
                    margin-bottom: 1rem;
                }

                .column-tuesday {
                    grid-column: 3;
                }
            
      </style>
    </div>
    <div
      className="client client-tuesday total-bottom"
    >
      <div
        className="number_of_photos"
      >
        -
      </div>
      <style
        jsx={true}
      >
        
            .client-tuesday {
                grid-column: 3
            }

            .client p {
                font-size: 0.8rem;
                margin: 0;
            }

            .client p span {
                font-size: 1rem;
                font-weight: 700;
            }

            .number_of_photos span:nth-child(2) {
                font-size: 0.7rem;
            }
        
      </style>
    </div>
    <div
      className="column column-wednesday"
    >
      wednesday
      <style
        jsx={true}
      >
        
                .column {
                    text-transform: lowercase;
                    font-variant: small-caps;
                    font-weight: 500;
                    font-size: 0.85rem;
                    border-bottom: 1px solid #f7f7f7;
                    margin-bottom: 1rem;
                }

                .column-wednesday {
                    grid-column: 4;
                }
            
      </style>
    </div>
    <div
      className="client client-wednesday total-bottom"
    >
      <div
        className="number_of_photos"
      >
        -
      </div>
      <style
        jsx={true}
      >
        
            .client-wednesday {
                grid-column: 4
            }

            .client p {
                font-size: 0.8rem;
                margin: 0;
            }

            .client p span {
                font-size: 1rem;
                font-weight: 700;
            }

            .number_of_photos span:nth-child(2) {
                font-size: 0.7rem;
            }
        
      </style>
    </div>
    <div
      className="column column-thursday"
    >
      thursday
      <style
        jsx={true}
      >
        
                .column {
                    text-transform: lowercase;
                    font-variant: small-caps;
                    font-weight: 500;
                    font-size: 0.85rem;
                    border-bottom: 1px solid #f7f7f7;
                    margin-bottom: 1rem;
                }

                .column-thursday {
                    grid-column: 5;
                }
            
      </style>
    </div>
    <div
      className="client client-thursday total-bottom"
    >
      <div
        className="number_of_photos"
      >
        -
      </div>
      <style
        jsx={true}
      >
        
            .client-thursday {
                grid-column: 5
            }

            .client p {
                font-size: 0.8rem;
                margin: 0;
            }

            .client p span {
                font-size: 1rem;
                font-weight: 700;
            }

            .number_of_photos span:nth-child(2) {
                font-size: 0.7rem;
            }
        
      </style>
    </div>
    <div
      className="column column-friday"
    >
      friday
      <style
        jsx={true}
      >
        
                .column {
                    text-transform: lowercase;
                    font-variant: small-caps;
                    font-weight: 500;
                    font-size: 0.85rem;
                    border-bottom: 1px solid #f7f7f7;
                    margin-bottom: 1rem;
                }

                .column-friday {
                    grid-column: 6;
                }
            
      </style>
    </div>
    <div
      className="client client-friday total-bottom"
    >
      <div
        className="number_of_photos"
      >
        -
      </div>
      <style
        jsx={true}
      >
        
            .client-friday {
                grid-column: 6
            }

            .client p {
                font-size: 0.8rem;
                margin: 0;
            }

            .client p span {
                font-size: 1rem;
                font-weight: 700;
            }

            .number_of_photos span:nth-child(2) {
                font-size: 0.7rem;
            }
        
      </style>
    </div>
    <div
      className="column column-saturday"
    >
      saturday
      <style
        jsx={true}
      >
        
                .column {
                    text-transform: lowercase;
                    font-variant: small-caps;
                    font-weight: 500;
                    font-size: 0.85rem;
                    border-bottom: 1px solid #f7f7f7;
                    margin-bottom: 1rem;
                }

                .column-saturday {
                    grid-column: 7;
                }
            
      </style>
    </div>
    <div
      className="client client-saturday total-bottom"
    >
      <div
        className="number_of_photos"
      >
        -
      </div>
      <style
        jsx={true}
      >
        
            .client-saturday {
                grid-column: 7
            }

            .client p {
                font-size: 0.8rem;
                margin: 0;
            }

            .client p span {
                font-size: 1rem;
                font-weight: 700;
            }

            .number_of_photos span:nth-child(2) {
                font-size: 0.7rem;
            }
        
      </style>
    </div>
    <div
      className="column column-sunday"
    >
      sunday
      <style
        jsx={true}
      >
        
                .column {
                    text-transform: lowercase;
                    font-variant: small-caps;
                    font-weight: 500;
                    font-size: 0.85rem;
                    border-bottom: 1px solid #f7f7f7;
                    margin-bottom: 1rem;
                }

                .column-sunday {
                    grid-column: 8;
                }
            
      </style>
    </div>
    <div
      className="client client-sunday total-bottom"
    >
      <div
        className="number_of_photos"
      >
        -
      </div>
      <style
        jsx={true}
      >
        
            .client-sunday {
                grid-column: 8
            }

            .client p {
                font-size: 0.8rem;
                margin: 0;
            }

            .client p span {
                font-size: 1rem;
                font-weight: 700;
            }

            .number_of_photos span:nth-child(2) {
                font-size: 0.7rem;
            }
        
      </style>
    </div>
    <style
      jsx={true}
    >
      
                    .WeekTable {
                        display: grid;
                        grid-auto-flow: column;
                        grid-template-columns: repeat(8, 1fr);
                    }

                    .WeekTable &gt; div {
                        padding: 0.5rem;
                    }

                    .WeekTable &gt; .odd {
                        background-color: #f7f7f7;
                    }

                    .WeekTable &gt; .total-bottom {
                        border-top: 2px solid #f7f7f7;
                    }

                    .WeekTable &gt; .total-top {
                        border-bottom: 2px solid #f7f7f7;
                    }
                
    </style>
  </div>,
]
`;
